<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>853</width>
      <height>682</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="accessibleName">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;hallo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="activationOrder">
     <enum>QMdiArea::CreationOrder</enum>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::SubWindowView</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsMovable">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::East</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionDatei_oder_Projekt_ffnen"/>
    <addaction name="separator"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichern_unter"/>
    <addaction name="separator"/>
    <addaction name="actionDrucken"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuErstellen">
    <property name="title">
     <string>Erstellen</string>
    </property>
    <addaction name="actionAlles_erstellen"/>
    <addaction name="actionProjekt_erstellen"/>
    <addaction name="actionProjekt_bereinigen"/>
    <addaction name="separator"/>
    <addaction name="actionProjektAusfuehren"/>
   </widget>
   <widget class="QMenu" name="menuDebuggen">
    <property name="title">
     <string>Debuggen</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Analyse</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionEinstellungen"/>
   </widget>
   <widget class="QMenu" name="menuFenster">
    <property name="title">
     <string>Fenster</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuErstellen"/>
   <addaction name="menuDebuggen"/>
   <addaction name="menuAnalyse"/>
   <addaction name="menuExtras"/>
   <addaction name="menuFenster"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionNeu">
   <property name="text">
    <string>Neu ..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDatei_oder_Projekt_ffnen">
   <property name="text">
    <string>Datei oder Projekt öffnen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="text">
    <string>Speichern unter..</string>
   </property>
   <property name="toolTip">
    <string>speichert eine Datei unter einen angegebenen Namen</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="text">
    <string>Drucken</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionAlles_erstellen">
   <property name="text">
    <string>Alles erstellen</string>
   </property>
  </action>
  <action name="actionProjektAusfuehren">
   <property name="text">
    <string>Ausführen</string>
   </property>
  </action>
  <action name="actionProjekt_erstellen">
   <property name="text">
    <string>Projekt erstellen</string>
   </property>
  </action>
  <action name="actionProjekt_bereinigen">
   <property name="text">
    <string>Projekt bereinigen</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>pressed()</signal>
  <signal>resized()</signal>
  <slot>on_MainWindow_resized()</slot>
 </slots>
</ui>
